content/blog/posts/osl-alumni-matthew-tesla.rst

```diff
@@ -0,0 +1,18 @@
+OSL Alumni Matthew Johnson on working at Tesla
+==============================================
+:date: 2019-03-07
+:slug: osl-alumni-matthew-johnson
+:author: Jerry Peng
+:img: matthew.jpg
+:tag: featured-stories
+
+Matthew Johnson is an OSL alumni who currently works at Tesla as a senior software engineer.
+He recently sat down with College of Engineering to talk about his experience working at Tesla.
+
+`YouTube Video: Graduate Matthew Johnson`_
+
+"Matthew Johnson is a veteran of the U.S Marine Corp Infantry and served as a police officer. 
+At Oregon State, he worked for the Open Source Lab and received a scholarship from CBT Nuggets. 
+He now works for Tesla as a senior software engineer."
```

Comment:
This seems repetitive to what's stated above and it's also surrounded with quotes. I'd rather see it incorporated above.